src/ExpandAnimations.bas

```diff
@@ -73,39 +88,24 @@ function renameAsExpanded(doc as Object)
   Endif
 
   sDocUrl = doc.getURL()
-  sDocPath = DirectoryNameoutofPath(sDocUrl, "/")
-  sDocFileName = FileNameoutofPath(sDocUrl, "/")
-  sDocFileNameExtension = GetFileNameExtension(sDocUrl)
   sDocFileNameWithoutExtension = GetFileNameWithoutExtension(sDocUrl, "/") 
-  
-  newUrlExpanded = sDocPath + "/" + sDocFileNameWithoutExtension + "-expanded.odp"
+
+  newUrlExpanded = "file:///tmp/" + sDocFileNameWithoutExtension + "-expanded.odp"
```

Comment:
@katz-hm, as you seem to be a Windows user (and developer?), could you file a PR to replace the hard-coded `/tmp` here with the content of an appropriate environment variable?
According to this official document, `%TEMP%` or `%TMP%` should be suitable candidates.
Only, this variable is not set, the code should fallback to the (Linux) default `/tmp`

Reply:
I'd love to contribute, but I completely lack BASIC skills. I tried reading the TEMP Env, but I think I need to replace backslashes by regular slashes it to form a valid file URI (not sure if that's all):

```
tempPath = Environ("TEMP")
newUrlExpanded = "file:///"+tempPath+"/" + sDocFileNameWithoutExtension + "-expanded.odp"
```

Resulting exception was

Message: SfxBaseModel::impl_store <file:///C:\Users\KATZ~1.BAS\AppData\Local\Temp/DijkstraAnimated-expanded.odp>

If I hardcode 
```
"file:///C:/Users/KATZ~1.BAS/AppData/Local/Temp/"+ sDocFileNameWithoutExtension + "-expanded.odp"
```
It works well. So if you could provide me with any source for the BASIC I need (or a solution to search and replace), I could prepare a PR, given that I find a way to write conditional code for the fallback, too.

Really proper way would be to additionally check for accessibility and fall back to the local folder, I think.

Reply:
I can try later this evening not prepare a PR.

Reply:
I'd be happy to test it.